.pre-commit-config.yaml

```diff
@@ -104,7 +104,7 @@ repos:
           - --settings=.
         exclude: /__init__\.py$
   - repo: https://github.com/acsone/setuptools-odoo
-    rev: 2.6.0
+    rev: 3.0.3
```

Comment:
Please don't modify here. The good method is to apply copier update in another PR

Reply:
@max3903 Click version should not be pinned. Black has to be updated. 

Please review this instead https://github.com/OCA/stock-logistics-warehouse/pull/1400/files

Reply:
@max3903 I will merge an update there that is less intrusive: #1418